EDDI/App.xaml.cs

```diff
@@ -1,9 +1,15 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Diagnostics;
 using System.Globalization;
+using System.IO;
+using System.Reflection;
+using System.Security.Permissions;
 using System.Threading;
 using System.Threading.Tasks;
 using System.Windows;
+using EddiCore;
+using EddiSpeechService;
```

Comment:
Adds some "usings" which are currently not used. Will these be used later or should these be cleaned up?
- using System.Collections.Generic;
- using System.IO;
- using System.Reflection;
- using System.Security.Permissions;
- using EddiSpeechService;

Reply:
Cleaned up and sorted.